misk-crypto/src/main/kotlin/misk/crypto/CryptoTestModule.kt

```diff
@@ -11,29 +12,79 @@ import com.google.crypto.tink.aead.AeadKeyTemplates
 import com.google.crypto.tink.mac.MacConfig
 import com.google.crypto.tink.aead.KmsEnvelopeAead
 import com.google.crypto.tink.mac.MacFactory
-import com.google.crypto.tink.mac.MacKeyTemplates
 import com.google.crypto.tink.signature.PublicKeySignFactory
 import com.google.crypto.tink.signature.PublicKeyVerifyFactory
 import com.google.crypto.tink.signature.SignatureConfig
-import com.google.crypto.tink.signature.SignatureKeyTemplates
 import com.google.inject.name.Names
 import misk.inject.KAbstractModule
 import javax.inject.Inject
 import javax.inject.Provider
 
 /**
  * This module should be used for testing purposes only.
- * It generates random keys for each key name specified in the configuration
+ * It uses the hardcoded keys defined in [TEST_AEAD_KEYSET], [TEST_MAC_KEYSET] and [TEST_SIGNATURE_KEYSET]
  * and uses [FakeKmsClient] instead of a real KMS service.
  *
  * This module **will** read the exact same configuration files as the real application,
  * but **will not** use the key material specified in the configuration.
- * Instead, it'll generate a random keyset handle for each named key.
+ * Instead, it'll generate a predefined [KeysetHandle] for each named key.
  */
 class CryptoTestModule(
   private val keyNames: List<Key>
 ) : KAbstractModule() {
 
+  private companion object {
+    /**
+     * Fake master key to be used instead of a real [com.google.crypto.tink.KmsClient]
+     */
+    val masterKey = FakeMasterEncryptionKey()
+    /**
+     * Hardcoded [Aead] [KeysetHandle] that's loaded and used by [AeadProvider]
+     */
+    const val TEST_AEAD_KEYSET = "{\n" +
```

Comment:
worth calling out how these were generated?

Reply:
They were generated like this:
```
val kh = KeysetHandle.generateKeyset(AeadKeyTemplates.AES_GCM256)
val output = ByteArrayOutputStream()
val writer = JsonKeysetWriter.withOutputStream(output)
kh.write(writer, FakeKmsClient().getAead())
output.toString()
```
This is mostly an internal thing and I shouldn't be exposed

Reply:
Yes, that's what it was like until now.
I want those keys to be permanent and hardcoded because if they're generated on the fly every time the app starts you'll get a different key - which means that every time you'd want to run your app locally, you'll have to delete all the data in your database from previous runs.